.github/workflows/build-deploy-hwp-frontend.yaml

```diff
@@ -54,21 +54,27 @@ jobs:
           cmd: |
             'version'
             
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
       - name: Deploy to DEV
         env:
           NAMESPACE: "${{ secrets.OPENSHIFT_DEV_NAMESPACE }}"
-          APPLICATION_NAME: "${{ secrets.FRONTEND_APPLICATION_NAME }}"
+          APPLICATION_NAME: "${{ secrets.API_APPLICATION_NAME }}"
```

Comment:
file is build-deploy-hwp-frontend but application name is API?

Reply:
copy pasta. thank you for catching that.